invenio_files_rest/config.py

```diff
@@ -22,12 +22,21 @@
 # waive the privileges and immunities granted to it by virtue of its status
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 
-"""Invenio File Rest module configuration file."""
+"""Invenio Files Rest module configuration file."""
 
 FILES_REST_STORAGE_CLASS_LIST = {
```

Comment:
IMHO it is not configuration but it should be defined as enum in models. Can admin define more storage classes? If yes, then we should have something like: `'S': 'invenio_files_rest.storage:Standard'`.